### ISSUE TITLE
UNKNOWN_USER in system.errors for SQL created users

### ISSUE THREAD

**Issue by den-crane** [ClickHouse/ClickHouse#26835]
```
21.8.1.7409

create user foo identified by 'secret';

clickhouse-client -u foo --password=secret

SELECT
    name,
    value,
    last_error_message
FROM system.errors
WHERE code = 192

┌─name─────────┬─value─┬─last_error_message──────────────────┐
│ UNKNOWN_USER │     1 │ There is no user `foo` in users.xml │
└──────────────┴───────┴─────────────────────────────────────┘

```

**Comment by alexey-milovidov** [comment]
@vitlibar Exceptions cannot be used in non-exceptional cases.

**Comment by etadelta222** [comment]
A bit more background on this. I am trying to connect to single node set-up clickhouse to Superset. I created a new user using SQL user management per Altinity blog. Gave the user read only permissions to database. Tried using `clickhouse+native://` URI to connect but it didn't work. Tried just `clickhouse://` URI and I'm able to connect, look at the db and the tables. However if I query any tables I am not getting any results. I queried the system.errors table and it returns result with no header. That's when I noticed the 'UNKNOWN_USER' error.
I used the 'clickhouse+native://` to connect to the clickhouse-public db without any issues ([blog](https://altinity.com/blog/visualizing-clickhouse-data-with-apache-superset-part-1-installation)).
